js/tabSlide.js

```diff
@@ -1,32 +1,64 @@
-var tabSlide = {};
 
-tabSlide.populateTab = function() {
-  $('projects').each(function() {
-    if (!$(this).hasClass('template')) {
+(function(module) {
 
-      var val = $(this).find('address a').text();
-      var optionTag = '<option value="' + val + '">' + val + '</option>';
-      $('#project-filter').append(optionTag);
 
-      val = $(this).attr('data-category');
-      optionTag = '<option value="' + val + '">' + val + '</option>';
-      if ($('#category-filter option[value="' + val + '"]').length === 0) {
-        $('#category-filter').append(optionTag);
+  var tabSlide = {};
+
+  tabSlide.funFacts = function() {
+    return Portfolio.projArray.map(function(portfolio){
+      return portfolio.title.length;
+    })
+    .reduce(function(a,b){
+      return a+b;
+    })
+    $('footer').append('<p>I have ' + funFacts() + ' projects!</p>') //not sure how to grab the number of projects I have needs more work
```

Comment:
The .append() is never going to fire, since it's after a return statement. You should move this line into a separate function - especially since you already have a call to funFacts() inside that function. (calling a function inside itself is called recursion and I'm pretty sure it's not what you meant to do here)